test/f90_correct/src/qc02.f08

```diff
@@ -0,0 +1,119 @@
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+!
+! test for exponentiation operation of real complex : complex(4) ^ complex(4)
+
+! not supported!
```

Comment:
what are the consequences of not being supported?

Reply:
Removed.